doc/varnish-configuration.rst

```diff
@@ -33,6 +33,18 @@ simple ACL, valid for all Varnish versions from 3 onwards, looks as follows:
     trigger invalidation are whitelisted here. Otherwise, lost cache invalidation
     requests will lead to lots of confusion.
 
+Advanced Varnish Configuration: xkey
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With Varnish 4.1 and higher you can optionally take advantage of official Varnish
+modules, specifically `xkey Varnish module`_. This module will provide better
+performance for tag handling as it adds secondary hashes to objects, and as
+it allows for grace handling by means of softpurge on tags.
```

Comment:
Since Varinsh 4.1 you can use the official [xkey Varnish module] for better performance with cache tags.
The module adds secondary hashes to objects, allowing to purge them specifically instead of using 
generic ``BAN`` instructions. This further allows grace handling by means of softpurge on tags.